docs/ephemeral_ssh_key_integration_plan.md

```diff
@@ -0,0 +1,94 @@
+# Ephemeral SSH Key Integration Plan for Nerdlog
```

Comment:
This doc should be not an integration plan, but an actual doc on how to use it. The integration plan, if we want to have it, should be posted as a PR description or something.